components/brave_private_new_tab_ui/common/pref_names.h

```diff
@@ -0,0 +1,22 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_PRIVATE_NEW_TAB_UI_COMMON_PREF_NAMES_H_
+#define BRAVE_COMPONENTS_BRAVE_PRIVATE_NEW_TAB_UI_COMMON_PREF_NAMES_H_
+
+class PrefRegistrySimple;
+
+namespace brave_private_new_tab::prefs {
+
+constexpr char kBravePrivateWindowDisclaimer[] =
```

Comment:
I think the naming of these constants has been confusing me - how about we match to the intention and the string? Something like `kBravePrivateWindowDisclaimerDismissed`?

Reply:
makes sense